x/ibc/03-connection/client/utils/utils.go

```diff
@@ -176,6 +177,23 @@ func QueryConnectionConsensusState(
 	return res, nil
 }
 
+// ParseClientState unmarshals a cmd input argument from a JSON string to a client state
+// If the input is not a JSON, it looks for a path to the JSON file
+func ParseClientState(cdc *codec.LegacyAmino, arg string) (clientexported.ClientState, error) {
```

Comment:
this should use proto

Reply:
I followed the convention of the other functions, not sure how to use proto in this context

Reply:
added to #6392 . We can deal with it later